sway-core/src/asm_generation/from_ir.rs

```diff
@@ -2300,7 +2329,7 @@ mod tests {
 
         let asm_script = format!("{}", asm);
         if asm_script != expected {
-            tracing::error!("{}", prettydiff::diff_lines(&expected, &asm_script));
+            print!("{}", prettydiff::diff_lines(&expected, &asm_script));
```

Comment:
I think the tracing subscriber is not initialized at the time of this line getting executed. This happened with the testing suite before (#1624), so `tracing::error` is not printed out, for forc-* parts of the repo this is not a problem since every part of the forc initializes the tracing subscriber but I think it is not clear where to initialize the subscriber for places like test-suite and sway-core so that is missed. I am opening up an issue for correcting this and will work on this quickly.

Reply:
That being said, I'm not sure why I changed it to `print!` instead of `println!` 😆 .. I'll change it.